### PR TITLE
update content of capp bilden

### PR DESCRIPTION
**Pull request by DavidFritsch**

### CODE REVIEW

**Comment by fatboypunk** [comment]
```
- ./build/capp-entwickeln/index.html
  *  Unexpected end tag : article (line 353)
  *  Unexpected end tag : div (line 406)
  *  Unexpected end tag : div (line 408)
  *  Unexpected end tag : section (line 354)
```

File: source/localizable/capp/_capp-entwickeln.de.erb [hunk 1]
```diff
@@ -49,48 +64,27 @@
...
-        <p>
-          <figure>
-            <%= image_tag "screenshots/eportfolio-2.png", title: "E-portfolio", alt: "E-portfolio", class: "center screenshot" %>
-            <figcaption>E-portfolio</figcaption>
-          </figure>
+        <p class="featured"><span class="star">*</span>
+          Ein E-Portfolio für die Personalentwicklung trägt dazu bei, jeden Mitarbeiter seinen Stärken gemäß am optimalen Platz einsetzen. Dabei unterstützt Sie "CAPP Entwickeln" mit einer Vielzahl hilfreicher Funktionen!
         </p>
 
       </div>
```
Comment (fatboypunk) [inline source/localizable/capp/_capp-entwickeln.de.erb]: I think we're not removing enough div's here

Comment (RonaldDijkstra) [reply source/localizable/capp/_capp-entwickeln.de.erb]: We're missing 1 opening tag: `<div class="row">` before the final div in the content. When added: 

> HTML-Proofer finished successfully.

**Comment by RonaldDijkstra** [comment]
Nice job @DavidFritsch

**Pull request merged**